includes/Core/Remote_Features/Remote_Features_Syncer.php

```diff
@@ -78,6 +88,20 @@ public function pull_remote_features() {
 
 		if ( ! is_wp_error( $features ) && is_array( $features ) ) {
 			$this->remote_features->set( $features );
+			$this->remote_features_last_sync->set( time() );
 		}
 	}
+
+	/**
+	 * Calls the CRON action.
+	 *
+	 * @since n.e.x.t
+	 */
+	public function pull_remote_features_fallback() {
+		check_ajax_referer( 'remote_features_fallback' );
+
+		do_action( Remote_Features_Cron::CRON_ACTION );
+
+		wp_send_json_success();
+	}
```

Comment:
As it is, this method could really go anywhere, but let's not add it here. Instead, let's co-locate it with the other fallback related implementation, and perhaps a reference to the callback, similar to how it was done for the CRON action without explicitly coupling one to another.